experiments/integrated/fluxnet/buildkite_test.jl

```diff
@@ -0,0 +1,13 @@
+ARGS = ["US-MOz"]
+include("run_fluxnet.jl")
+include("inputs_dataframe.jl")
+include("climalsm_output_dataframe.jl") 
+
+if isdir("figures")
```

Comment:
I think the test runs from home directory of the repo. I would suggest `mkdir` of the whole path, using `joinpath`, to combine the path to the repo (gotten using `pkgdir`) and then combining with the full path from the repo to the figures directory, I think this would be `experiments/integrated/fluxnet/$(site_ID)/figures`

An example of this is [here](https://github.com/CliMA/ClimaLSM.jl/blob/c730511de8f04d99eed9add6d352193783a745a3/experiments/integrated/fluxnet/run_fluxnet.jl#L34)

Reply:
thanks! I'll try that